project.clj

```diff
@@ -10,18 +10,23 @@
   :dependencies [[org.clojure/clojure "1.7.0"]
 
                  ;; begin version conflict resolution dependencies
-                 [clj-time "0.10.0"]
+                 [clj-time "0.11.0"]
                  [commons-codec "1.9"]
                  [org.clojure/tools.macro "0.1.5"]
+                 [org.clojure/tools.reader "1.0.0-alpha1"]
```

Comment:
fwiw, file-sync and puppetserver have a dep on the alpha version. but moving to the beta instead of the alpha seems like a good move to me.